lib/sort_by_length.rb

```diff
@@ -1,7 +1,24 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(1)
 def sort_by_length(my_sentence)
```

Comment:
Nice bubblesort.  Technically since you're splitting the string into an array... this has O(n) space complexity.